packages/react-core/src/components/Page/examples/CenteredSection.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import {
+  Page,
+  Masthead,
+  MastheadToggle,
+  MastheadMain,
+  MastheadBrand,
+  MastheadContent,
+  PageSidebar,
+  PageSection,
+  PageToggleButton,
+  Toolbar,
+  ToolbarContent,
+  ToolbarItem,
+  Card,
+  CardBody
+} from '@patternfly/react-core';
+import BarsIcon from '@patternfly/react-icons/dist/esm/icons/bars-icon';
+
+export const CenterAlignedPageSection: React.FunctionComponent = () => {
+  const [isNavOpen, setIsNavOpen] = React.useState(true);
+
+  const onNavToggle = () => {
+    setIsNavOpen(!isNavOpen);
+  };
+  const headerToolbar = (
+    <Toolbar id="centered-toolbar">
+      <ToolbarContent>
+        <ToolbarItem>header-tools</ToolbarItem>
+      </ToolbarContent>
+    </Toolbar>
+  );
+
+  const Header = (
+    <Masthead>
+      <MastheadToggle>
+        <PageToggleButton
+          variant="plain"
+          aria-label="Global navigation"
+          isNavOpen={isNavOpen}
+          onNavToggle={onNavToggle}
+          id="centered-nav-toggle"
+        >
+          <BarsIcon />
+        </PageToggleButton>
+      </MastheadToggle>
+      <MastheadMain>
+        <MastheadBrand href="https://patternfly.org" target="_blank">
+          Logo
+        </MastheadBrand>
+      </MastheadMain>
+      <MastheadContent>{headerToolbar}</MastheadContent>
+    </Masthead>
+  );
+  const Sidebar = <PageSidebar nav="Navigation" isNavOpen={isNavOpen} id="main-padding-sidebar" />;
+  return (
+    <Page header={Header} sidebar={Sidebar}>
```

Comment:
The original example actually didn't have a sidebar passed in. Maybe we should keep it that way?

Reply:
We discussed this via slack and we agreed on maybe it being an oversight so I added the sidebar. But no problem for me to fix it. 😊

Reply:
nah - sounds good for me! :)